core/src/main/java/nu/yona/server/subscriptions/service/UserAnonymizedCacheService.java

```diff
@@ -0,0 +1,41 @@
+package nu.yona.server.subscriptions.service;
+
+import java.util.UUID;
+
+import org.springframework.cache.annotation.CacheConfig;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.CachePut;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import nu.yona.server.exceptions.InvalidDataException;
+import nu.yona.server.subscriptions.entities.UserAnonymized;
+
+@CacheConfig(cacheNames = "usersAnonymized")
+@Service
+class UserAnonymizedCacheService
```

Comment:
Omit ``Cache`` from the name. For consumers, it's simply the service that deals with ``UserAnonymized``.